### PR TITLE
[TIMOB-19763] (5_1_X) Support item-specific peek for previewContext on ListView

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/TIMOB-19763

### CODE REVIEW

File: apidoc/Titanium/UI/iOS/PreviewAction.yml [hunk 1]
```diff
@@ -40,9 +40,8 @@ events:
     summary: Fired when the device detects a click against a preview action.
     description: |
         This event provides different properties depending on the view the <Titanium.UI.iOS.PreviewContext>
-        is assigned to. For example, the <Titanium.UI.ListView> and <Titanium.UI.TableView> provide the
-        additional properties `itemIndex` and `sectionIndex` to identify the previewed row and
-        section, respectively.
+        is assigned to. For example, the <Titanium.UI.ListView> provides the
+        additional properties `itemIndex` and `sectionIndex` to identify the previewed row and section, respectively.
```
Comment (FokkeZB) [inline apidoc/Titanium/UI/iOS/PreviewAction.yml]: You missed `itemId` here.

File: apidoc/Titanium/UI/iOS/PreviewContext.yml [hunk 2]
```diff
@@ -171,12 +232,19 @@ examples:
...
-                Ti.API.warn("sectionIndex: " + e.sectionIndex); // Index of the ListView section
-                Ti.API.warn("itemIndex: " + e.itemIndex); // Index of the ListView row
-                detailWindow.open();
-            }
+            contentHeight: 300 // When unspecified, we use the available height
+        });
+
+        // Fired after peeking the preview
+        // Use this event to configure the preview depending on the sectionIndex / itemIndex
+        context.addEventListener("peek", function(e) {
```
Comment (FokkeZB) [inline apidoc/Titanium/UI/iOS/PreviewContext.yml]: Would be good to actually do something useful here. What would you do in a real app?

Comment (hansemannn) [inline apidoc/Titanium/UI/iOS/PreviewContext.yml]: The docs sample are supposed to be as simple as possible. If users want more detailed examples, they normally check out the sample apps or just try out their own ideas

File: apidoc/Titanium/UI/iOS/PreviewAction.yml (hunk 1 above)
Comment (hansemannn) [inline apidoc/Titanium/UI/iOS/PreviewAction.yml]: We have it, was just undocumented and is now included.

**Comment by pec1985** [comment]
PR reviewed. **APPROVED!**

**Pull request merged**